entrypoint.sh

```diff
@@ -55,6 +55,62 @@ declare -r AUTOSCALING_GROUP_NAME=$(
         || echo "AutoDetectionFailed"
     )
 
+declare -r  TERMINATE_STALE_EC2=${TERMINATE_STALE_EC2:-true}
+if [ "$TERMINATE_STALE_EC2" = true ]; then
+    __info_log "Set TERMINATE_STALE_EC2=true"
+else
+    __info_log "Set TERMINATE_STALE_EC2=false"
+fi
+
+declare -r  ECS_CLUSTER="$(curl -s $ECS_CONTAINER_METADATA_URI/task | jq -r .Cluster)"
+if [ -z "$ECS_CLUSTER" ]; then
+    __error_log "Unable to fetch the name of the cluster."
+    exit 1
+fi
+declare -r  ATTRIBUTES_FOR_STALE_EC2="${ATTRIBUTES_FOR_STALE_EC2:-}"
+declare -ir MAX_ATTRIBUTES=${MAX_ATTRIBUTES:-10}  # AWS Restriction
+declare     ATTRIBUTES_FOR_AWSCLI=""
```

Comment:
Use array instead, then remove the below codes(L101-L105) for formatting to join with a space.

Reply:
d92eb07